sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,137 @@
+        git log -1 --pretty=%h > "${jobs_repo}/ORIGIN_COMMIT"
+        source hack/lib/init.sh
+        os::build::rpm::format_nvra > "${jobs_repo}/ORIGIN_BUILT_VERSION"
+    - type: "script"
+      title: "build junitreport"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/junitreport hack/env make build WHAT=tools/junitreport
+    - type: "script"
+      title: "build service-catalog user-broker image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        REGISTRY=openshift/ NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog user-broker-image
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        unset GOPATH; cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/build-go.sh
+    - type: "script"
+      title: "run Service Catalog unit tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog test-unit
+    - type: "script"
+      title: "run Service Catalog integration tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo yum -y install etcd
+        # this strips the -i arg to "go test". can be dropped once PR is merged:
+        # https://github.com/kubernetes-incubator/service-catalog/pull/1566 (and vendored in rebase)
+        sed -i 's/\(go test.*\)\( -i \)\(.*\)/\1 \3/' cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/test/integration.sh
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog scBuildImageTarget=.scBuildImage test-integration
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py service-catalog
+    - type: "script"
+      title: "build Service Catalog E2E test binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog  bin/e2e.test
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc cluster up --version=latest --service-catalog
+    - type: "script"
+      title: "Run Service Catalog E2E"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo chmod a+rw /var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        export KUBECONFIG=/var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
```

Comment:
Though you may need the above configuration for oc commands.

Reply:
FYI, I just had to do a change in service catalog tests around our use of kubeconfig.  See https://github.com/openshift/origin/issues/18722.  The work around is simple (use oc config view --raw > tmp and use the tmp file for kubeconfig).

Reply:
Unrelated to the above, soon I plan on updating the way origin is built for catalog. And you'll probably want to copy such that origin changes won't be able to break catalog (and therefore cause your job to fail).